src/index.ts

```diff
@@ -79,12 +90,17 @@ async function typeLines(
   lines: string[],
   contentElement: HTMLElement,
   characterDelay: number,
-  lineDelay: number
+  lineDelay: number,
+  reduceMotion: boolean
 ): Promise<void> {
   for (let lineIndex = 0; lineIndex < lines.length; lineIndex++) {
-    for (const character of lines[lineIndex].split('')) {
-      await wait(characterDelay)
-      contentElement.innerHTML += character
+    if (reduceMotion) {
+      contentElement.append(lines[lineIndex])
+    } else {
```

Comment:
Do we need `reduceMotion` if `characterDelay` is already `0` in the case that the user prefers reduced motion?

Reply:
Yes, because otherwise there is still a bit of a delay in the lines appearing one-by-one because we would have been appending each character one by one (even though there is a delay of `0`). It looks as if it's typing, but just really really fast, and it doesn't have quite the same affect as the all the characters appearing at one time.

Reply:
Maybe an approach that could help consolidate / simplify the logic is if we replace
`if (reduceMotion)` with `if (characterDelay === 0)`, so that way we don't have to feed `reduceMotion` as an extra param, and we have the parity of using both `characterDelay` and `lineDelay`. What do you think?

Reply:
Love that idea! I like how it consolidates the logic, and I think it makes sense that we'd have the same behavior whether the user has prefers reduced motion or we've passed in `0` for character delay and line delay.